.github/workflows/dockerhub_e2e_all_base_runtime.yml

```diff
@@ -25,3 +28,4 @@ jobs:
           push: true
           tags: |
             atsigncompany/noports_e2e_all_base_runtime:latest
+            atsigncompany/noports_e2e_all_base_runtime:gha${{ github.run_number }}
```

Comment:
We normally tag with a version or GitHub Actions run number so we know where an image came from.